Makefile

```diff
@@ -259,8 +259,9 @@ build-docker: sha = $(shell git describe --always --abbrev=12)
 build-docker:
 	docker build -t launcher-build --build-arg gitver=$(sha) .
 
+build-dockerfake: version = $(shell git describe --always --tags --abbrev=12)
```

Comment:
If this works with docker, 👍

Reply:
It's working all right (as far as I can tell) with `make containers` -- it's pretty much the same as the `sha = ` above